src/ops/query.rs

```diff
@@ -20,6 +21,8 @@ impl ExecuteQuery {
         query_ofs: i32,
         gas_limit: u64,
     ) -> Result<(), VMError> {
+        trace!("executing 'query'");
+
```

Comment:
in all ops/* tracing messages I'd add text that it is a "host" function, also, I'd make them all trace level, I'd avoid other levels unless there is a good reason

Reply:
Will be done.